### PR TITLE
adding BFS & DFS implementation in c++

### LINKED ISSUE #94: Addition of Important Graph Algorithms implemented in C++
This is to include some of the most important graph algorithms implemented using C++.
If it's fine with you I can creatw a PR so that you can merge it to the project. 
Thank You!

### PR DESCRIPTION
**Pull request by Jappan07**
This is for the addition of Depth First Search and Breadth First Search Graph Algorithm implementation in c++.

The two files for the algorithms are being added to the C++ folder in the repo.
Please review my PR and hopefully merge it to the main project which will resolve the issue number #94 

To run the code all one need is a C++ compiler.
#94 

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

# Checklist:

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: C++/bfs_traversal.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+    cout << "Adjacency list:" << endl;
+    g.print();
+    cout << "Breadth First Search Traversal Applied:" << endl;
+    g.bfs(0);
+
+
+    return 0;
+}
+
```
Comment (kaustubhgupta) [inline C++/bfs_traversal.cpp]: Get rid of these extra lines

File: C++/dfs_traversal.cpp [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    cout << "Adjacency list:" << endl;
+    g.print();
+    cout << "Depth First Search Traversal Applied:" << endl;
+    g.dfs("a");
+    cout << endl;
+
+
+    return 0;
+}
+
```
Comment (kaustubhgupta) [inline C++/dfs_traversal.cpp]: Get rid of these extra lines

File: C++/bfs_traversal.cpp [hunk 3]
```diff
@@ -0,0 +1,98 @@
+#include<iostream>
```
Comment (kaustubhgupta) [inline C++/bfs_traversal.cpp]: Delete .DS_store file

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by Jappan07** [comment]
Thank you for the suggesting the changes. I have rectified them. Please review and merge the PR.

**Review by kaustubhgupta — APPROVED** [review]

**Review by YashSaxena9 — CHANGES_REQUESTED** [review]
Code reviewed, please remove .DS_Store binary file.

**Comment by Jappan07** [comment]
Done. Removed .DS_Store file.

**Pull request merged**